### PR TITLE
Updated CRD and CLP mission trees

### PR DESCRIPTION
**Pull request by That-Guy-Github1**
conversion of CRD and CLP mission trees with mild changes made to latter.

### CODE REVIEW

File: history/provinces/6222 - Formless Keystone.txt
Comment (Hecatia-Lapislazuli) [inline history/provinces/6222 - Formless Keystone.txt:1]: If to become uncolonized, should have other features to match

File: localisation/touhou_events_l_english.yml [hunk 1]
```diff
@@ -2156,6 +2156,11 @@
  flavor_clp.15.a: "Let them come! We will decimate them!"
  flavor_clp.15.a.tt: "Every owned province with the §YLunarian§! culture will get '§YLunarians' Last Stand§!' for §Y30§! years, giving the following effect:\nYears of Separatism: §G-30§!"
  flavor_clp.15.b: "We should negotiate with them before they pick up their weapons."
+ flavor_clp.16.t: "The Clown's advance"
```
Comment (Hecatia-Lapislazuli) [inline localisation/touhou_events_l_english.yml]: Title capitalization
```suggestion
 flavor_clp.16.t: "The Clown's Advance"
```

File: events/flavorCLP.txt [hunk 2]
```diff
@@ -1002,4 +1002,163 @@ country_event = {
...
+				}
+				add_permanent_claim = ROOT
+			}
+		sun_garden_state = {
+				limit = {
+					NOT = { is_core = ROOT }
+					NOT = { is_permanent_claim = ROOT }
+				}
+				add_permanent_claim = ROOT
+			}
```
Comment (Hecatia-Lapislazuli) [inline events/flavorCLP.txt]: Indentation is messy here
```suggestion
		nightmarish_halls_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		gengetsu_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		lotus_road_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		panzer_graveyard_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		rika_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		rika_state_2 = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		ai_chance = { 
			factor = 100
		}
	}

	#March west
	option = {
		name = flavor_clp.16.b
		sun_garden_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		nameless_hill_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		sunflower_state_5 = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		mugenkan = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		rehearsal_hall = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		phantom_land_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		ai_chance = { 
			factor = 0
		}
	}
	#March south
	option = {
		name = flavor_clp.16.c
		concert_hall_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		forest_of_werewolf = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		canal_of_willows = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		raiko_seki_state2 = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		rehearsal_hall = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
		sun_garden_state = {
			limit = {
				NOT = { is_core = ROOT }
				NOT = { is_permanent_claim = ROOT }
			}
			add_permanent_claim = ROOT
		}
```

File: missions/CLP_Missions.txt [hunk 3]
```diff
@@ -86,31 +90,78 @@ clp_slot1_branch = {
 
 		provinces_to_highlight = {
 			OR = {
-				province_id = 5320
-				province_id = 5322
+				province_id = 5872 # replaced because it felt like it made more geographical sense, willing to take any and all inputs on what provinces should be used
 			}
```
Comment (Hecatia-Lapislazuli) [inline missions/CLP_Missions.txt:94]: If kept at only one province, redundant OR block. Unsure with requested feedback for now, I'll check later.

File: missions/CLP_Missions.txt [hunk 4]
```diff
@@ -702,39 +781,191 @@ clp_slot3_branch = {
...
+						area = concert_hall_state
+						area = forest_of_werewolf
+						area = canal_of_willows
+						area = raiko_seki_state2
+						area = rehearsal_hall
+						area = phantom_land_state
 					}
 				}
-				amount = 6
+			}
```
Comment (Hecatia-Lapislazuli) [inline missions/CLP_Missions.txt]: I think is just missing an indent?

File: missions/CLP_Missions.txt [hunk 5]
```diff
@@ -765,16 +1002,19 @@ clp_slot3_branch = {
 
 	CLP_clowpiece_superiority = {
 		icon = mission_touhou_clownpiece
-		required_missions = { CLP_the_great_human_pranking }
-		position = 6
+		required_missions = { 
+		CLP_the_great_human_pranking 
+		CLP_liberation_of_fairy
+		}
```
Comment (Hecatia-Lapislazuli) [inline missions/CLP_Missions.txt]:
```suggestion
		required_missions = { CLP_the_great_human_pranking CLP_liberation_of_fairy }
```

File: missions/CRD_Missions.txt [hunk 6]
```diff
@@ -163,18 +150,14 @@ crd_scarlet_diplo_slot = {
...
 					type = all
-				}
-				Scarlet_mountain = {
-					country_or_non_sovereign_subject_holds = ROOT
-					type = all
-				}
-				Scarlett_domain = {
+					OR = { 
+					area = darkglade_state  
 					country_or_non_sovereign_subject_holds = ROOT
```
Comment (Hecatia-Lapislazuli) [inline missions/CRD_Missions.txt]:
```suggestion
					OR = { 
						area = darkglade_state  
						country_or_non_sovereign_subject_holds = ROOT
```

File: missions/CRD_Missions.txt [hunk 7]
```diff
@@ -220,18 +202,14 @@ crd_scarlet_diplo_slot = {
 				}
 				AND = {
 					NOT = { exists = SCR }
-					Scarlet_island = {
+					misty_lake_region = {
+						OR = { 
+						area = darkglade_state  
 						owned_by = ROOT
```
Comment (Hecatia-Lapislazuli) [inline missions/CRD_Missions.txt]:
```suggestion
						OR = { 
							area = darkglade_state  
							owned_by = ROOT
```

File: missions/CRD_Missions.txt [hunk 8]
```diff
@@ -220,18 +202,14 @@ crd_scarlet_diplo_slot = {
...
-						type = all
-					}
-					Scarlet_mountain = {
-						owned_by = ROOT
-						type = all
 					}
-					Scarlett_domain = {
-						owned_by = ROOT
-						type = all
+
```
Comment (Hecatia-Lapislazuli) [inline missions/CRD_Missions.txt]:
```suggestion
```

File: missions/CRD_Missions.txt [hunk 9]
```diff
@@ -400,14 +392,14 @@ crd_conquest_religion_slot = {
...
-				province_id = 5051
+				province_id = 5275
 				province_id = 5005
 			}
 			NOT = { country_or_non_sovereign_subject_holds = ROOT }
 		}
 		
 		trigger = {
-			owns_or_non_sovereign_subject_of = 5051
+			owns_or_non_sovereign_subject_of = 5275#previous province already owned by CRD
```
Comment (Hecatia-Lapislazuli) [inline missions/CRD_Missions.txt]:
```suggestion
			owns_or_non_sovereign_subject_of = 5275 #previous province already owned by CRD
```

File: missions/CRD_Missions.txt [hunk 10]
```diff
@@ -532,67 +524,217 @@ crd_conquest_religion_slot = {
...
-						country_or_non_sovereign_subject_holds = ROOT
-						religion = ROOT
-					}
-					Road_to_hell_state = {
+					west_former_hell_region = {
 						type = all
 						country_or_non_sovereign_subject_holds = ROOT
 						religion = ROOT
 					}
+
```
Comment (Hecatia-Lapislazuli) [inline missions/CRD_Missions.txt]:
```suggestion
```

File: missions/CRD_Missions.txt [hunk 11]
```diff
@@ -856,23 +991,21 @@ crd_imperial_slot = {
...
 			owns_or_non_sovereign_subject_of = 4983
 			num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
 				OR = {
-					area = Higan_state
-					area = Eastern_higan
-					area = Higan_southern_coast
-					area = Western_higan
+					region = lower_hell_region
+				region = sanzu_river_region
+				region = upper_hell_region
```
Comment (Hecatia-Lapislazuli) [inline missions/CRD_Missions.txt]:
```suggestion
					region = lower_hell_region
					region = sanzu_river_region
					region = upper_hell_region
```

**Review by Hecatia-Lapislazuli — CHANGES_REQUESTED** [review]

**Review by Hecatia-Lapislazuli — APPROVED** [review]

File: missions/CLP_Missions.txt (hunk 3 above)
Comment (ComradeFlan) [reply missions/CLP_Missions.txt:94]: Uusally not an issue

**Pull request merged**